include/deal.II/hp/fe_collection.h

```diff
@@ -100,6 +100,12 @@ namespace hp
     FECollection<dim, spacedim> &
     operator= (FECollection<dim,spacedim> &&) = default; // NOLINT
 
+    /**
+     * Comparison operator. All stored Finite Element objects are compared in order.
+     */
+    bool
+    operator== (const FECollection<dim,spacedim> &fe_collection) const;
```

Comment:
while there, could you please add `operator !=`? That would make code below read better.